Feature/tests energy account statement import file 2

AlvaroGRuiz opened the pull request:
OP#1723

AlvaroGRuiz committed:
[IMP] ✨ New tests for module energy_account_statement_import_file

Author: Alvaro Garcia <alvaro.garcia@somit.coop>

AlvaroGRuiz committed:
[IMP] ✨ Now you can import 2 lines equals on the same file

Author: Alvaro Garcia <alvaro.garcia@somit.coop>

juan-pe reviewed (approved):
Great @AlvaroGRuiz ! no more duplicated lines anymore! 🎉 🎉

AlvaroGRuiz committed:
[REL] 🏗️  Bump energy_account_statement_import_file to v16.0.0.1.1

Author: Alvaro Garcia <alvaro.garcia@somit.coop>
Author: Juan Pedro Sánchez <juanpe.sanchez@somenergia.coop>
